spec/lib/evss/pciu/service_spec.rb

```diff
@@ -65,4 +65,56 @@
       end
     end
   end
+
+  describe '#post_primary_phone' do
+    let(:phone) { build(:phone_number, :nil_effective_date) }
+
+    context 'when successful' do
+      it 'returns a status of 200' do
+        VCR.use_cassette('evss/pciu/post_primary_phone') do
```

Comment:
Are these matching on just the request body, or is it matching on request method and header as well?

Reply:
The whole kit and caboodle :)